railties/lib/rails/generators/app_base.rb

```diff
@@ -298,6 +298,10 @@ def rails_version_specifier(gem_version = Rails.gem_version)
         end
       end
 
+      def npm_version
```

Comment:
I'm not sure if this should be `:nodoc:` or `:doc:` but I guess we don't want to expose it as public API yet?

Also, I wasn't sure if we should add the origin of this method which has a much better description:
https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/tasks/release.rb#L26-L36

Reply:
Note to self: https://guides.rubyonrails.org/api_documentation_guidelines.html#method-visibility

It seems this whole class is [nodoc'd](https://edgeapi.rubyonrails.org/classes/Rails/Generators/AppBase.html) so probably adding these comments to the method won't make a difference.

Reply:
Fixed in 9080c8f5b5